src/frontend/components/SelectContent/index.tsx

```diff
@@ -0,0 +1,430 @@
+import React, { useEffect } from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Grid,
+  Heading,
+  Image,
+  Layer,
+  ResponsiveContext,
+  Stack,
+  Text,
+  TextInput,
+  Tip,
+} from 'grommet';
+
+import { DocumentMissing, DocumentUpload, Monitor } from 'grommet-icons';
+import { Document } from '../../types/file';
+import { Video } from '../../types/tracks';
+import { Nullable } from '../../utils/types';
+import styled from 'styled-components';
```

Comment:
If you take a look in the other components or js module, we first create a group with external dependencies and then the modules we developed.